### ISSUE TITLE
Visualization: Help creating accuracy and loss plots?

### ISSUE THREAD

**Issue by EricaXia** [HazyResearch/snorkel#1333]
Firstly, thank you Snorkel team for all your hard work.

Is it possible to plot accuracy and loss over epochs, as the end extraction model trains, with Tensorboard or other tools?  Additionally, how can we create ROC curves or Precision-Recall curves, similar to sklearn functions? So far, I’ve trained a logistic regression model as per the tutorial [here](https://github.com/HazyResearch/snorkel/blob/master/tutorials/workshop/Workshop_6_Advanced_Grid_Search.ipynb), as my discriminative model. 

I'm a newbie and would like some help going in the right direction. Thanks

**Comment by vincentschen** [comment]
Hi @EricaXia— glad to hear you're trying the tutorial! 

In general, we don't support ML visualization tools because there are already a number of fantastic tools out there!

For plotting loss over epochs, I'd recommend [Tensorboard](https://www.tensorflow.org/guide/summaries_and_tensorboard) (in v0.9, we'll support this more natively in our logging tools). For ROC/PRC, you could plot them using `sklearn` after outputing the raw predictions using the [`marginals` method](https://github.com/HazyResearch/snorkel/blob/master/snorkel/learning/tensorflow/noise_aware_model.py#L274). 

Hope that helps!
